### ISSUE TITLE
[documentation] react-query legacyMode description is incorrect

### ISSUE THREAD

**Issue by rliljest** [dotansimha/graphql-code-generator#9324]
### Describe the bug

According to the [website documentation](https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-react-query#legacymode), the default value of `legacyMode` is `true`. Looking at the [4.0.0 CHANGELOG](https://github.com/dotansimha/graphql-code-generator-community/blob/63a5322bd8db4c123bd2906551a3a1fabb32b2ae/packages/plugins/typescript/react-query/CHANGELOG.md#400) it says that the value is `false` by default (which is correct)

### Your Example Website or App

N/A

### Steps to Reproduce the Bug or Issue

Go to the [website documentation](https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-react-query#legacymode) and see the incorrect information

### Expected behavior

I would expect the website documentation to match the actual behavior of the `legacyMode` option

### Platform

macOS

**Comment by charpeni** [comment]
Correct, I checked the code and it defaults to `false`: https://github.com/dotansimha/graphql-code-generator-community/blob/bf71aee287c39618c3a3f34132257ad7cacc6234/packages/plugins/typescript/react-query/src/visitor.ts#L87.

We should update the documentation accordingly.
